### DISCUSSION TITLE
Metallic effect on the recorded audio.

### DISCUSSION THREAD

**Discussion by alexgrusu** [SimformSolutionsPvtLtd/audio_waveforms#257] [General]
I noticed that the recorded audio has some metal effect on it.

I tried different configurations but nothing changed.

I ended up trying a different package which doesn't seem to affect the recorded audio.

Has anyone noticed this effect so far?

**Comment by yile-li** [comment]
May I ask which package you ended up using? I saw your comment before implementing it so I'm not sure if it affects the audio. Appreciate it!
